loc/classes/archer.js

```diff
@@ -9,6 +9,7 @@ export default {
     es: 'Arquero',
     fr: 'Archer',
     pt: 'Arqueiro'
+    de: 'Bogenschütze',
```

Comment:
```suggestion
    de: 'Bogenschütze'
```

Reply:
I thought you had to have the comma there. I saw everyone else had it by theirs.

Reply:
No, you don't put commas at the end of the last translation, however you do put them at the end of the other translations.